### ISSUE TITLE
Bug - Table - columns that are not sortable should not have `aria-sort`

### ISSUE THREAD

**Issue by rhamilto** [patternfly/patternfly-react#11023]
**Describe the problem**
Per the latest W3C specifications and accessibility guidelines, the `aria-sort` attribute should only have values of “ascending”, “descending”, or “other” to indicate the sort order.  If sorting does not exist for the column, [no `aria-sort` attribute should be added](https://adrianroselli.com/2021/04/sortable-table-columns.html#Basics).  https://github.com/patternfly/patternfly-react/blob/1850cf48db06c9c4459b76a3f4af355671cf9f3e/packages/react-table/src/components/Table/utils/decorators/sortable.tsx#L55 currently outputs a value of "none" for columns that are not sortable but should not.

**How do you reproduce the problem?**
In OpenShift Management Console, upgrade `axe-core` and `cypress-axe` to the latest versions, run `other-routes.cy.ts`, and note the `aria-sort="none"` accessibility failures.

**Expected behavior**
No `aria-sort="none"` accessibility issues should be present after upgrading `axe-core` and `cypress-axe` to the latest versions in OpenShift Management Console.

**Is this issue blocking you?**
Yes.  We cannot [upgrade `axe-core` and `cypress-axe` in OpenShift Management Console](https://issues.redhat.com/browse/OCPBUGS-41826) as a result of this bug, which in turn is blocking the ability to upgrade `monaco-editor`.

**Screenshots**
<img width="2128" alt="Screenshot 2024-09-18 at 5 07 11 PM" src="https://github.com/user-attachments/assets/dbff7921-e164-4b88-93ce-0922aa6ee21b">

cc: @logonoff

**Comment by rhamilto** [comment]
Any update on this?  It is blocking several bugs in OpenShift console.

**Comment by tlabaj** [comment]
closed by #11144
